### PR TITLE
Apply offsets to sky positions

### PR DESCRIPTION
**Pull request by cquiroz**

### CODE REVIEW

File: bundle/edu.gemini.p2checker/src/main/scala/edu/gemini/p2checker/rules/ghost/GhostRule.scala [hunk 1]
```diff
@@ -63,18 +63,38 @@ object GhostRule extends IRule {
...
+            base,
+            t1.coordinates(when).map(_.offset(SRIFU1SeparationOffset.p.toAngle, SRIFU1SeparationOffset.q.toAngle)),
+            t2.coordinates(when).map(_.offset(SRIFU2SeparationOffset.p.toAngle, SRIFU2SeparationOffset.q.toAngle)))
         case GhostAsterism.TargetPlusSky(t, s, _)               =>
-          checkBoth(ctx, node, base, t.coordinates(when).map(_.offset(SRIFU1SeparationOffset.p.toAngle, SRIFU1SeparationOffset.q.toAngle)), Some(s.coordinates))
+          checkBoth(ctx,
+            node,
+            base,
+            t.coordinates(when).map(_.offset(SRIFU1SeparationOffset.p.toAngle, SRIFU1SeparationOffset.q.toAngle)),
+            Some(s.coordinates.offset(SRIFU2SeparationOffset.p.toAngle, SRIFU2SeparationOffset.q.toAngle)))
```
Comment (swalker2m) [inline bundle/edu.gemini.p2checker/src/main/scala/edu/gemini/p2checker/rules/ghost/GhostRule.scala:82]: Code looks good but remind me what the separation offset is again?  If you apply it to both IFUs for the check is that not the same as not applying it to either for the check?

Comment (swalker2m) [reply bundle/edu.gemini.p2checker/src/main/scala/edu/gemini/p2checker/rules/ghost/GhostRule.scala:82]: Wait I see, they are different offsets for IFU1 and IFU2.

**Review by swalker2m — APPROVED** [review]
:+1:

File: bundle/edu.gemini.p2checker/src/main/scala/edu/gemini/p2checker/rules/ghost/GhostRule.scala (hunk 1 above)
Comment (cquiroz) [reply bundle/edu.gemini.p2checker/src/main/scala/edu/gemini/p2checker/rules/ghost/GhostRule.scala:82]: Yes, this PR is about applying the separations to the sky positions

**Pull request merged**